api/src/main/resources/Reference_Application_Concepts-23.xml

```diff
@@ -115,34 +115,42 @@
   <concept concept_id="162821" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-01-06 22:32:39.0" version="" changed_by="1" date_changed="2015-01-06 22:32:47.0" retired_by="1" uuid="162821AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="162822" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-01-06 22:33:22.0" version="" changed_by="1" date_changed="2015-01-06 22:33:29.0" retired_by="1" uuid="162822AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
   <concept concept_id="163139" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-12-13 15:00:18.0" version="" changed_by="1" date_changed="2015-12-13 15:02:03.0" retired_by="1" uuid="163139AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="165095" retired="false" datatype_id="3" class_id="7" is_set="false" creator="1" date_created="2018-04-18 14:34:20.0" version="" changed_by="1" date_changed="2018-04-18 14:34:30.0" retired_by="1" uuid="165095AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept_numeric concept_id="1343" units="cm" precise="true"/>
```

Comment:
Maybe this needs to be added @dkayiwa ?

Reply:
What happens if it is not added?

Reply:
To be honest, I don't really have much of a clue about the implications of having added(or not having added) this line. The reason being that I just copied the output from the tool as mentioned in the Wiki

Reply:
Am not specifically referring to this line. I mean all changes in this file. Did you take a look at each of them?